### PR TITLE
Splitted off the main part into a .py file to make it easier to use the macro in scripts

### PR DESCRIPTION
**Pull request by chbergmann**
Thank you for creating a pull request to contribute to FreeCAD-macros!
To integrate your macro please make sure the following steps are complete:

- [ ] Please check this box if you're not submitting a new macro.   
- [ ] Are you submitting a new macro ?  
- [ ] Have you followed the ['How to submit a macro'](../README.md#how-to-submit-a-macro) section of the README.md ?  
- [ ] Your macro has a [Description](../README.md#macro-description) in its header.  
- [ ] Your macro has a [CamelCase name](../README.md#camelcase-macro-name).  
- [ ] Your macro is named [appropriately](../README.md#macro-name-specifics).  
- [ ] Your macro contains a [Metadata section](../README.md#macro-metadata) that immediately follows the header description.  
- [ ] Your macro is Python3/Qt5 compliant and tested on the latest FreeCAD stable and development releases.  
- [ ] You're including documentation on how your macro works (bonus: screenshots and/or video on the Wiki)  
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)  
- [ ] Commit message is titled in the following way `[MacroName] Short description`.
- [ ] Optional, write or update the changelog in the macro, from latest to oldest.

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

**Comment by luzpaz** [comment]
It would also be beneficial to add and icon to the `__icon__` in the metadata portion of the macro

File: ImportExport/ObjectsToPython.FCMacro [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 __License__ = 'LGPL-3.0-or-later'
 __Web__ = ''
 __Wiki__ = 'README.md'
 __Name__ = 'Objects To Python'
 __Comment__ = 'Exports objects from a FreeCAD project to a python script'
 __Author__ = 'Christi'
-__Icon__ = ''
-__Help__ = ''
-__Status__ = ''
+__Icon__ = 'ObjectsToPython/toconsole.svg'
```
Comment (galou) [inline ImportExport/ObjectsToPython.FCMacro:11]: Can you please rename `ObjectsToPython/toconsole.svg` to `ObjectsToPython.svg`?

File: ImportExport/ObjectsToPython.FCMacro [hunk 2]
```diff
@@ -1,19 +1,19 @@
 # -*- coding: utf-8 -*-
 
-__Version__ = '0.6'
-__Date__ = '2021-11-14'
+__Version__ = '0.7'
+__Date__ = '2021-07-17'
```
Comment (galou) [inline ImportExport/ObjectsToPython.FCMacro:4]: Please use today's date.

File: TechDraw/LasercutterSVGExport.FCMacro [hunk 3]
```diff
@@ -205,117 +39,20 @@ class LasercutterTechdrawExport():
...
-        gui.doCommand("from LasercutterTechdrawExport import LasercutterTechdrawExport")
-        
+        gui.doCommand("from LasercutterSVGExport import LasercutterTechdrawExport")
+
         gui.doCommand("lcparts = []")
         selection = gui.Selection.getSelectionEx()
         for sel in selection:
             gui.doCommand("lcparts.append(FreeCAD.ActiveDocument.getObject('%s'))"%(sel.ObjectName))
             
         gui.doCommand("LasercutterTechdrawExport.makeLasercutterTechdrawExport(lcparts, BeamWidth = %f, doc = App.activeDocument())"%self.form.doubleSpinBox.value())
```
Comment (galou) [inline TechDraw/LasercutterSVGExport.FCMacro:52]: Do you know `gui.doCommand(f'LasercutterTechdrawExport.makeLasercutterTechdrawExport(lcparts, BeamWidth={self.form.doubleSpinBox.value()}, doc=FreeCAD.activeDocument())')`?

I mainly wanted to mention formatted strings but I changed a few other details according to Python standard and consistency between the use of `FreeCAD` and `App`.

**Review by galou — CHANGES_REQUESTED** [review]
Thanks for the changes. Please consider my comments about cosmetic things.

File: ImportExport/ObjectsToPython.FCMacro [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
-__Icon__ = ''
-__Help__ = ''
-__Status__ = ''
+__Icon__ = 'ObjectsToPython.svg'
+__Help__ = 'ObjectsToPython/README.md'
+__Status__ = 'Alpha'
 __Requires__ = 'FreeCAD >= 0.19'
 __Communication__ = 'https://forum.freecadweb.org/viewtopic.php?f=22&t=34612'
-__Files__ = 'ObjectsToPython/README.md,ObjectsToPython/ObjectsToPython.ui'
+__Files__ = 'ObjectsToPython/README.md,ObjectsToPython/ObjectsToPython.ui,ObjectsToPython/toconsole.svg'
```
Comment (galou) [inline ImportExport/ObjectsToPython.FCMacro:16]: `ObjectsToPython.svg` it meant in place of `ObjectsToPython/toconsole.svg`.

File: TechDraw/LasercutterSVGExport.FCMacro [hunk 5]
```diff
@@ -4,199 +4,33 @@ __Name__ = 'LasercutterSVGExport'
 __Comment__ = 'Creates contour lines for all selected objects and arranges them on a TechDraw page'
 __Author__ = 'christi'
 __Version__ = '0.7'
```
Comment (galou) [inline TechDraw/LasercutterSVGExport.FCMacro:6]: `__Version__ = '0.7.0'`

File: ImportExport/ObjectsToPython.FCMacro [hunk 6]
```diff
@@ -1,19 +1,19 @@
 # -*- coding: utf-8 -*-
 
-__Version__ = '0.6'
-__Date__ = '2021-11-14'
+__Version__ = '0.7'
```
Comment (galou) [inline ImportExport/ObjectsToPython.FCMacro:3]: `__Version__ = 0.7.0`

**Review by galou — CHANGES_REQUESTED** [review]
Sorry that I missed a few things in the last review. Thanks for your patience.

**Comment by galou** [comment]
Thanks!

**Pull request merged**